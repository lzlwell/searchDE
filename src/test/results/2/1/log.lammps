LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26863 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -273.76936   -273.76936      17.1317           10           50 
       1   -580.12201   -580.12201      17.1317           10           50 
       2   -742.08081   -742.08081      17.1317           10           50 
       3     -817.393     -817.393      17.1317           10           50 
       4   -881.12845   -881.12845      17.1317           10           50 
       5    -936.4866    -936.4866      17.1317           10           50 
       6   -985.34536   -985.34536      17.1317           10           50 
       7   -1027.1164   -1027.1164      17.1317           10           50 
       8   -1061.4291   -1061.4291      17.1317           10           50 
       9   -1088.7889   -1088.7889      17.1317           10           50 
      10   -1110.4484   -1110.4484      17.1317           10           50 
Loop time of 0.366659 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -273.769358526      -1088.7888696     -1110.44836384
  Force two-norm initial, final = 2884.77 148.979
  Force max component initial, final = 1798.44 110.458
  Final line search alpha, max atom move = 0.000625068 0.0690436
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.144047 (39.2864)
Neigh time (%) = 0.219146 (59.7684)
Comm  time (%) = 0.00144839 (0.395024)
Outpt time (%) = 0.000137329 (0.0374542)
Other time (%) = 0.00187993 (0.512719)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2396 ave 2396 max 2396 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46652 ave 46652 max 46652 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46652
Ave neighs/atom = 155.507
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
