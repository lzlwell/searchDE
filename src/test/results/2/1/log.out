LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26863 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -273.76936   -273.76936      17.1317           10           50 
       1   -580.12201   -580.12201      17.1317           10           50 
       2   -742.08081   -742.08081      17.1317           10           50 
       3     -817.393     -817.393      17.1317           10           50 
       4   -881.12845   -881.12845      17.1317           10           50 
       5    -936.4866    -936.4866      17.1317           10           50 
       6   -985.34536   -985.34536      17.1317           10           50 
       7   -1027.1164   -1027.1164      17.1317           10           50 
       8   -1061.4291   -1061.4291      17.1317           10           50 
       9   -1088.7889   -1088.7889      17.1317           10           50 
      10   -1110.4484   -1110.4484      17.1317           10           50 
Loop time of 0.366659 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -273.769358526      -1088.7888696     -1110.44836384
  Force two-norm initial, final = 2884.77 148.979
  Force max component initial, final = 1798.44 110.458
  Final line search alpha, max atom move = 0.000625068 0.0690436
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.144047 (39.2864)
Neigh time (%) = 0.219146 (59.7684)
Comm  time (%) = 0.00144839 (0.395024)
Outpt time (%) = 0.000137329 (0.0374542)
Other time (%) = 0.00187993 (0.512719)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2396 ave 2396 max 2396 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46652 ave 46652 max 46652 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46652
Ave neighs/atom = 155.507
Neighbor list builds = 10
Dangerous builds = 10
